docs/getting-started/01-overview.md

```diff
@@ -0,0 +1,81 @@
+
+- [Redis addon](https://github.com/kyma-project/addons/tree/master/addons/redis-0.0.3) is basically a bundle of two Redis services available in two plans: `micro` and `enterprise`. You will connect it to Kyma through Helm Broker and expose it in the Kyma cluster as an addon under Service Catalog. The Redis service represents an open-source, in-memory data structure store, used as a database, cache and message broker. For the purpose of these guides, you will use the `micro` plan with in-memory storage to demonstrate how it can replace the default memory of the microservice.
+
+- [Commerce mock](https://github.com/SAP-samples/xf-addons/tree/master/addons/commerce-mock-0.1.0) is to act as a sample external and monolithic application which we want to extend with Kyma. It is based on the [Varkes](https://github.com/kyma-incubator/varkes) project and is also available in the form of an addon. It will simulate how you can pair an external application with Kyma and expose the application's APIs and events. In these guides, you will use its `order.deliverysent.v1` event type to trigger both the microservice and the Function.
+
+## Steps
+
+These guides cover the following steps:
+
+1. [Create a Namespace](#getting-started-create-a-namespace) (`orders-service`) in which you will deploy all resources.
+2. [Create a microservice](#getting-started-deploy-a-microservice) by deploying `orders-service` on the cluster in the `orders-service` Namespace.
+3. [Expose the microservice](#getting-started-expose-the-microservice) through the APIRule CR on HTTP endpoints. This way it will be available for other services outside the cluster so you could retrieve data from it and send sample orders to it. Since the microservice uses in-memory storage, when you send a sample order to its endpoint, the order details will be gone after you restart the microservice.
+4. [Add the Redis service](#getting-started-add-the-redis-service) as an addon.
+5. [Create the Redis service instance](#getting-started-create-a-service-instance-for-the-redis-service) in the Namespace (ServiceInstance CR) so you can bind it with the microservice and Function.
+6. [Bind the microservice to the Redis service](#getting-started-bind-the-redis-service-instance-to-the-microservice) by creating ServiceBinding and ServiceBindingUsage CRs. Send a sample order to its endpoint - since it now uses the Redis storage, the order details will not be gone after you restart the microservice.
+7. [Connect Commerce mock](#getting-started-connect-an-external-application) as the external application.
+8. [Trigger the microservice](#getting-started-trigger-the-microservice-with-an-event) to react to the `order.deliverysent.v1` event from the Commerce mock. Send the event and see if the microservice reacts to it by saving its details in the Redis database.
```

Comment:
```suggestion
8. [Trigger the microservice](#getting-started-trigger-the-microservice-with-an-event) to react to the `order.deliverysent.v1` event from the Commerce mock. Send the event, then see if the microservice reacts to it and saves its details in the Redis database.
```
`by saving` made me think I'm supposed to save it and thus check.

I also changed the first `and` to `then` to avoid double `and`, but if you don't like it, feel free to modify/skip it.

I did the same in line 77.